mesh_segmenter/src/mesh_segmenter.cpp

```diff
@@ -33,28 +31,34 @@ std::vector<vtkSmartPointer<vtkPolyData> > MeshSegmenter::getMeshSegments()
   vtkSmartPointer<vtkPolyData> input_copy = vtkSmartPointer<vtkPolyData>::New();
   input_copy->DeepCopy(input_mesh_);
   std::vector<vtkSmartPointer<vtkPolyData> > meshes;
-  for(int i = 0; i < included_indices_.size(); ++i)
+  //TODO: Figure out why the last one fails for some meshes (and then remove the -1)
+  for (int i = 0; i < included_indices_.size() - 1; ++i)
```

Comment:
All of your for and if statements now have an extra space prior to the open parenthesis. What style guide dictates this? I don't dislike it. I just want to know if I need to start doing this and also looking for it in my reviews.